pkg/netpol/connlist/connlist_test.go

```diff
@@ -1139,4 +1140,229 @@ var goodPathTests = []struct {
 		testDirName:   "netpol_named_port_test_2",
 		outputFormats: []string{output.DefaultFormat},
 	},
+	// tests with adminNetworkPolicy
+	{
+		testDirName:   "anp_test1_deny_traffic_at_cluster_level",
+		outputFormats: []string{output.TextFormat},
+	},
+	{
+		testDirName:   "anp_test2_allow_traffic_at_cluster_level",
+		outputFormats: []string{output.TextFormat},
+	},
+	{
+		testDirName:   "anp_test3_pass_traffic",
+		outputFormats: []string{output.TextFormat},
+	},
+	{
+		// Should Deny traffic from slytherin to gryffindor and
+		// Deny traffic to slytherin from gryffindor respecting ANP with priority 50, ignoring ANP with priority 60
+		testDirName:   "anp_test_4",
+		outputFormats: ValidFormats,
```

Comment:
please use consistent test names, rename `anp_test1_deny...` to be `anp_test_1_deny...` (also for anp_test2 and anp_test3)

Reply:
done